pages/common/samtools.md

```diff
@@ -0,0 +1,36 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
```

Comment:
would "sequencing (genomics) data" work better as "gene sequencing data" (or some variation thereof if that isn't technically correct)?